_pages/language-selection.md

```diff
@@ -33,6 +33,16 @@ be major reasons to select one of these languages for your project. We feel
 relatively comfortable supporting these languages, but if all other factors
 are equal, we encourage selecting from our first tier.
 
+## Frequently-used frameworks
```

Comment:
Wasn't sure where to put this—open to ideas.

Reply:
I'm good with this – I can add links to the other language-specific frameworks that already exist in the guide in another pass.

I think after our guild conversation today, this page should be more about selecting ecosystems and a framework section is in keeping with that.